### PR TITLE
update Hyundai to use new DBC updates

### PR DESCRIPTION
**Pull request by xps-genesis**
PR for DBC change

https://github.com/commaai/opendbc/pull/284

### CODE REVIEW

**Comment by xps-genesis** [comment]
@adeebshihadeh , can you check why the check is failing? this is part of the new hyundai dbc change.

File: selfdrive/car/hyundai/carstate.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def update(self, cp, cp_cam):
         ret.gearShifter = GearShifter.unknown
     # Gear Selecton - This is only compatible with optima hybrid 2017
     elif self.CP.carFingerprint in FEATURES["use_elect_gears"]:
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/carstate.py:90]: Update `use_elect_gears` to match the new signal name

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ class Buttons:
   "use_elect_gears": [CAR.KIA_OPTIMA_H, CAR.IONIQ_EV_LTD, CAR.KONA_EV, CAR.IONIQ],
 
   # these cars use the FCA11 message for the AEB and FCW signals, all others use SCC12
-  "use_fca": [CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ, CAR.KONA, CAR.KONA_EV, CAR.KIA_FORTE, CAR.PALISADE],
+  "use_fca": [CAR.SONATA, CAR.ELANTRA, CAR.ELANTRA_GT_I30, CAR.KIA_STINGER, CAR.IONIQ_EV_LTD, CAR.KONA, CAR.KONA_EV, CAR.KIA_FORTE, CAR.PALISADE],
```
Comment (adeebshihadeh) [inline selfdrive/car/hyundai/values.py:218]: This change should be made in another PR.

**Comment by techasit14607** [comment]
Let's go to Thailand. Meet me and Samartchai Boonnoon

File: selfdrive/car/hyundai/carstate.py (hunk 1 above)
Comment (xps-genesis) [reply selfdrive/car/hyundai/carstate.py:89]: i did not understand? you want to me change name from use_elect_gears to use_ev_pc5 ?

File: selfdrive/car/hyundai/values.py (hunk 2 above)
Comment (xps-genesis) [reply selfdrive/car/hyundai/values.py]: reverted. possibly not an issue

File: selfdrive/car/hyundai/carstate.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/hyundai/carstate.py:89]: yeah, something like `use_ev_gears`

**Comment by adeebshihadeh** [comment]
Done in #1968
